source/fundamentals/builders/filters.txt

```diff
@@ -11,3 +11,549 @@ Filters Builders
    :class: singlecol
 
 .. _filters-builders:
+
+Overview 
+--------
+In this guide, we show you how to use **builders** to specify
+**filters** for your queries.
+
+Builders are classes provided by the MongoDB Java driver that allow you
+to use the builder pattern to interact with your MongoDB instance. The
+builders provided by the MongoDB Java driver can improve the readability
+of your code.  See the :doc:`Builders
+</fundamentals/builders/>` page for a list of Builders available in the
+Java driver. 
+
+
+Filters are the operations MongoDB uses to limit your results to what
+you want to see. Some examples of results from queries with filters are: 
+
+- Items that cost $0 to $25
+- A hotel with amenities that include an indoor swimming pool and free parking
+- A food critic review that mentions "spicy" 
+
+This guide shows you how to use builders with examples of the following
+types of operators: 
+
+- :ref:`Comparison <comparison>`
+- :ref:`Logical <logical>`
+- :ref:`Arrays <arrays>`
+- :ref:`Elements <elements>`
+- :ref:`Evaluation <evaluation>`
+- :ref:`Bitwise <bitwise>`
+- :ref:`Geospatial <geospatial>`
+
+The :java-core-api:`Filters <com/mongodb/client/model/Filters.html>`
+class provides static factory methods for all the MongoDB query
+operators. Each method returns an instance of the :ref:`Bson <bson>`
+type, which you can pass to any method that expects a query filter. 
+
+.. tip::
+
+   For brevity, you may choose to import all methods of the
+   :java-core-api:`Filters <com/mongodb/client/model/Filters.html>`
+   class statically: 
+
+   .. code-block:: java
+
+      import static com.mongodb.client.model.Filters.*;
+
+   The examples below assume this static import.
+
+The Filter examples in this guide use the following sample collections:
+
+Collection: ``paint_purchases``
+
+.. code-block:: json
+
+   {"_id": 1, "color": "red", "qty": 5, "vendor": ["A"]}
```

Comment:
Nitpick: I think I mentioned this in a comment in an earlier review of this PR about consistency of spacing in JSON. Sometimes you have spaces padding the "{" and the first field and spaces padding the "}" field. We should stay consistent.

In case it's helpful, here's a doc that provides some info on how our team decided to do javascript/json formatting:
https://docs.google.com/spreadsheets/d/1vxDasa0S2yrpXfTJkH8VooYBIasiHdOIotQOx_b8xfc/edit#gid=0
I think there's another source for this info, but can't locate it at the moment.

Applies to all the json instances.